dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/controllers/WindowController.java

```diff
@@ -71,21 +84,57 @@ private void exitAction(ActionEvent e) {
 
 	@SuppressWarnings("unused") @FXML
 	private void zoomInAction(ActionEvent e) {
-		strainView.zoomIn();
+		if (strainView != null) {
```

Comment:
You get NPs when you wish to zoom in before the strainview is created (in the welcome screen).